portfolio/src/main/webapp/styles/style.css

```diff
@@ -0,0 +1,149 @@
+/* General Elements */
+body {
+  background-color: #02003C;
+  color:rgb(160, 160, 160);
+}
+
+#header {
+  padding-top: 8vh
+}
+ 
+.content {
+  padding-top: 2vh;
+  padding-left: 12vw;
+  z-index:0;
+}
+ 
+.section {
+  padding-top: 30vw;
+}
+ 
+img {
+  height: 10%;
+  width: 10%;
+}
+ 
+/* Navigation Bar */
+#outter-nav {
+  top: 0;
+  width: 100vw;
+  position: fixed;
+  background-color: #02003C;
+  color: #55FBDC;
+  padding: 2vmin;
+  padding-top: 3vmin;
+ 
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
```

Comment:
Please sort these to alphabetical order and fix the rest of the file . Re-https://google.github.io/styleguide/htmlcssguide.html#CSS_Formatting_Rules.

Reply:
Done.